doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1355 @@
+      * :c:macro:`SO_RAI` - Changed from ``61`` to ``1061``.
+      * :c:macro:`SO_IPV6_DELAYED_ADDR_REFRESH` - Changed from ``62`` to ``1062``.
+
+Thingy:53: Matter weather station
+---------------------------------
+
+* Updated by enabling Link Time Optimization (LTO) by default for the ``release`` configuration.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Bluetooth samples
+-----------------
+
+* Added
+
+  * Support for the ``nrf54l15dk/nrf54l05/cpuapp`` and ``nrf54l15dk/nrf54l10/cpuapp`` board targets in the following samples:
+
+    * :ref:`central_and_peripheral_hrs`
+    * :ref:`central_bas`
+    * :ref:`bluetooth_central_hids`
+    * :ref:`bluetooth_central_hr_coded`
+    * :ref:`bluetooth_central_dfu_smp`
+    * :ref:`central_uart`
+    * :ref:`multiple_adv_sets`
+    * :ref:`peripheral_bms`
+    * :ref:`peripheral_cgms`
+    * :ref:`peripheral_cts_client`
+    * :ref:`peripheral_gatt_dm`
+    * :ref:`peripheral_hids_keyboard`
+    * :ref:`peripheral_hr_coded`
+    * :ref:`peripheral_mds`
+    * :ref:`peripheral_nfc_pairing`
+    * :ref:`peripheral_rscs`
+    * :ref:`peripheral_status`
+    * :ref:`shell_bt_nus`
+    * :ref:`ble_throughput`
+
+  * The Advertising Coding Selection feature to the following samples:
+
+    * :ref:`bluetooth_central_hr_coded`
+    * :ref:`peripheral_hr_coded`
+
+* Updated:
+
+  * The configurations of the non-secure ``nrf5340dk/nrf5340/cpuapp/ns`` board target in the following samples to properly use the TF-M profile instead of the predefined minimal TF-M profile:
+
+    * :ref:`bluetooth_central_hids`
+    * :ref:`peripheral_hids_keyboard`
+    * :ref:`peripheral_hids_mouse`
+
+    This change results from the Bluetooth subsystem transition to the PSA cryptographic standard.
+    The Bluetooth stack can now use the PSA crypto API in the non-secure domain as all necessary TF-M partitions are configured properly.
+
+  * The configurations of the following samples by increasing the main thread stack size (:kconfig:option:`CONFIG_MAIN_STACK_SIZE`) to prevent stack overflows:
+
+    * :ref:`bluetooth_central_hids`
+    * :ref:`peripheral_hids_keyboard`
+    * :ref:`peripheral_hids_mouse`
+
+    This change results from the Bluetooth subsystem transition to the PSA cryptographic API.
+
+  * The following samples to use LE Secure Connection pairing (:kconfig:option:`CONFIG_BT_SMP_SC_PAIR_ONLY`).
+
+    * :ref:`peripheral_gatt_dm`
+    * :ref:`peripheral_mds`
+    * :ref:`peripheral_cts_client`
+
+* :ref:`direct_test_mode` sample:
+
+  * Added:
+
+    * Loading of radio trims and a fix of a hardware errata for the nRF54H20 SoC to improve the RF performance.
+    * Support for the :ref:`nRF2220 front-end module <ug_radio_fem_nrf2220ek>`.
+    * Workaround for the hardware errata HMPAN-216 for the nRF54H20 SoC.
+
+* :ref:`central_uart` sample:
+
+  * Added reconnection to bonded devices based on their address.
+
```

Comment:
Please add the following note that describes the change done in the following PR:

https://github.com/nrfconnect/sdk-nrf/pull/21844

```suggestion

* :ref:`peripheral_hids_keyboard` sample:

  * Fixed the issue with the :kconfig:option:`CONFIG_NFC_OOB_PAIRING` Kconfig option that is defined at the sample level and could not be enabled due to the unmet dependency on the :kconfig:option:`CONFIG_HAS_HW_NRF_NFCT` Kconfig option.
    The issue is resolved by enabling the ``nfct`` node in the sample devicetree configuration, which sets the :kconfig:option:`CONFIG_HAS_HW_NRF_NFCT` Kconfig option to the expected value.
```

Reply:
Added